### DISCUSSION TITLE
Difficulty in taking the thigh, for example, in SPHinxsys

### DISCUSSION THREAD

**Discussion by Mirzaei97** [Xiangyu-Hu/SPHinXsys#442] [Ideas]
Hello 
good time
With the first recipe on the Sphinxsys site to create your own cases, the vtp file for example 15 is not created
Can you guide me? To make it

**Comment by Xiangyu-Hu** [comment]
I am not quite understand what do you mean?
Could you specify a bit more so that I can help?
Thanks.

**Reply by Mirzaei97** [reply]
Dear Xiangyu

I hope all is fine. Thank you for your reply. It seems that the .vtp file is needed to display results in Paraview. In SPHinxsys, example 15 (aortic-valve 2D) there isn't a .vtp file. 

Regards
Mirzaei
